wolfssl/wolfcrypt/dilithium.h

```diff
@@ -726,6 +726,7 @@ int wc_dilithium_export_public(dilithium_key* key, byte* out, word32* outLen);
 #ifdef WOLFSSL_DILITHIUM_PRIVATE_KEY
 WOLFSSL_API
 int wc_dilithium_export_private(dilithium_key* key, byte* out, word32* outLen);
+#define wc_dilithium_export_private_only    wc_dilithium_export_private
```

Comment:
Test case needs added if we are going to maintain this function signature. Please add a wc_dilithium_export_private_only call to one of our Dilithium tests.